components/tikv_util/src/lib.rs

```diff
@@ -121,28 +120,6 @@ pub trait AssertSend: Send {}
 
 pub trait AssertSync: Sync {}
 
-pub fn limit_size<T: Message + Clone>(entries: &mut Vec<T>, max: u64) {
```

Comment:
I'm not following the thread before; Is there any reason why you removing it?

Reply:
This function is not used in `tikv`, and it has been implemented in [raft-rs](https://github.com/tikv/raft-rs/blob/afabefa23196a4a23832add0087f5a522d8ccc3b/src/util.rs#L48).

Reply:
Ok, I got it.